### PR TITLE
l10n: Improved grammar

### PR DESCRIPTION
**Pull request by rakekniven**
Reported at Transifex.

Signed-off-by: rakekniven <2069590+rakekniven@users.noreply.github.com>

### CODE REVIEW

File: iOSClient/Supporting Files/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -588,7 +588,7 @@
 "_password_pdf_error_"          = "Wrong password";
 "_error_download_photobrowser_" = "Error: Unable to download photo";
 "_good_morning_"                = "Good morning";
-"_good_noon_"                   = "Good noon";
+"_good_noon_"                   = "Good day";
```
Comment (Valdnet) [inline iOSClient/Supporting Files/en.lproj/Localizable.strings:591]: Why "Good day" if the text string appears exactly at 12 o'clock?
https://github.com/nextcloud/ios/blob/17443651c8b978eef74b046241a72ebd936d5476/Widget/Nextcloud/NextcloudData.swift#L62

**Comment by Valdnet** [comment]
I would not use this phrase at present. Immediately after 12 o'clock I would give "Good afternoon".

But if we want to highlight a greeting between 12:00 to 12:59, "Good day" is actually the more popular 👍.

To keep the code tidy, I would change this value: "_good_noon_" to "_good_day_".

**Review by Valdnet — APPROVED** [review]

**Comment by marinofaggiana** [comment]
Changed in hardcode. thanks

**Comment by marinofaggiana** [comment]
@rakekniven 👍🏻 

- "_good_noon_"                   = "Good noon";
+ "_good_noon_"                   = "Good day";

**Pull request closed without merging**